subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -711,17 +711,17 @@ void ull_conn_iso_ticker_cb(uint32_t ticks_at_expire, uint32_t ticks_drift,
 		 * has been reached, and offset calculated.
 		 */
 		if (cis->lll.handle != 0xFFFF && cis->lll.active) {
-			cis->lll.event_count += (lazy + 1U);
+			cis->lll.event_count_prepare += (lazy + 1U);
```

Comment:
`event_count_prepare` is initialized to `LLL_CONN_ISO_EVENT_COUNT_MAX` but unconditionally incremented here. 

Does this not risk overflowing the counter? Is that intentional?

Reply:
yes, intentional to rollover and take the value `0` on the very first (and corresponding in case of skipped) ISO interval event.

Reply:
Can I bother you to add a comment stating that? 

Generally, unless completely obvious, I do prefer when overflows are intentional that they are documented as well, so this won't be considered a bug by someone else :)